magpie/run/magpie-run-project-tensorflow-horovod

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+########################################################################
+#  Project Magpie. For details, see https://github.com/llnl/magpie.
+#
+#  Copyright (C) 2019 Intel Corporation. All rights reserved.
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License version
+#  2 as published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  Aleksander Kantak <aleksander.kantak@intel.com>
+########################################################################
+
+source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-log
+
+Magpie_run_start_tensorflow_horovod () {
+
+    if [ "${TENSORFLOW_HOROVOD_SETUP}" != "yes" ]
```

Comment:
Your code is correct, but it took me a second to grok.  Could we stick to the code style
in the rest of the Magpie.  This pattern from tensorflow should be good.  Dunno if you want to output a small header for users?  Dunno if that'd be useful or not.

```
    if [ "${TENSORFLOW_SETUP}" == "yes" ] && [ "${magpie_run_prior_startup_successful}" == "true" ]
    then
        if [ "${MAGPIE_CLUSTER_NODERANK}" == "0" ]
        then
            echo "*******************************************************"
            echo "*"
            echo "* Tensorflow Information"
            echo "*"
            echo "* Running with python = ${MAGPIE_PYTHON}"
            echo "*******************************************************"
        fi

        magpie_run_tensorflow_should_be_torndown=1
        magpie_run_tensorflow_setup_successful=1
    else
        magpie_run_zookeeper_should_be_torndown=0
        magpie_run_zookeeper_setup_successful=1
    fi
....

Reply:
In quoted code, and also in code on the master branch, you are using zookeepers variables in 'else'. I will fix it in the commit.

Reply:
whoops, thanks.  I will update in master.